eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/OperationMethodProviders/ResourceOperationMethodProvider.cs

```diff
@@ -124,91 +118,33 @@ private TryExpression BuildTryExpression()
             return new TryExpression(tryStatements);
         }
 
-        private CatchExpression BuildCatchExpression(VariableExpression scopeVariable)
-        {
-            return Catch(
-                Declare<Exception>("e", out var exceptionVariable),
-                [
-                    scopeVariable.Invoke(nameof(DiagnosticScope.Failed), exceptionVariable).Terminate(),
-                    Throw()
-                ]);
-        }
-
-        private MethodBodyStatement BuildRequestContextInitialization(ParameterProvider cancellationTokenParameter, out VariableExpression contextVariable)
-        {
-            var requestContextParams = new Dictionary<ValueExpression, ValueExpression>
-            {
-                { Identifier(nameof(RequestContext.CancellationToken)), cancellationTokenParameter }
-            };
-            return Declare(
-                "context",
-                typeof(RequestContext),
-                New.Instance(typeof(RequestContext), requestContextParams),
-                out contextVariable);
-        }
-
-        private MethodBodyStatement BuildHttpMessageInitialization(VariableExpression contextVariable, out VariableExpression messageVariable)
-        {
-            var requestMethod = _serviceMethod.GetCorrespondingRequestMethod(_resourceClientProvider);
-
-            var arguments = PopulateArguments(
-                requestMethod.Signature.Parameters,
-                contextVariable);
-
-            var invokeExpression = _resourceClientProvider
-                .GetRestClientField()
-                .Invoke(requestMethod.Signature.Name, arguments);
-
-            return Declare(
-                "message",
-                typeof(HttpMessage),
-                invokeExpression,
-                out messageVariable);
-        }
-
         private IReadOnlyList<MethodBodyStatement> BuildClientPipelineProcessing(
             VariableExpression messageVariable,
             VariableExpression contextVariable,
             out VariableExpression responseVariable)
         {
-            var statements = new List<MethodBodyStatement>();
             var responseType = _convenienceMethod.Signature.ReturnType!.UnWrapAsync();
-            VariableExpression declaredResponseVariable;
-            var pipelineInvoke = _isAsync ? "ProcessMessageAsync" : "ProcessMessage";
 
+            // Check if the return type is a generic Response<T> or just Response
             if (!responseType.Equals(typeof(Response)))
```

Comment:
responseType.IsGenericType

Reply:
replace the original condition with `responseType.IsGenericType` will lead to generator crash.